src/test/java/com/teamscale/upload/NativeImageIT.java

```diff
@@ -476,4 +475,4 @@ private Arguments withInput(String input) {
 		}
 	}
 
-}
+}
```

Comment:
All you need for a test on demo instance is

`@Test
	public void successfulUploadWithRepository() {
		ProcessUtils.ProcessResult result = runUploader(new Arguments().withRepository("cqse/teamscale-upload");
		assertThat(result.exitCode).describedAs("Stderr and stdout: " + result.stdoutAndStdErr).isZero();
	}`